runtime/lua/vim/lsp/client.lua

```diff
@@ -804,6 +805,10 @@ end
 ---
 --- @param force? boolean
 function Client:stop(force)
+  if self.is_stopped() then
+    return
+  end
+  self.is_stopping = true
   local rpc = self.rpc
 
   if rpc.is_closing() then
```

Comment:
since is_stopped() checks both, we can simplify this code.

```suggestion
  if self.is_stopped() then
    return
  end
  
  self.is_stopping = true
  local rpc = self.rpc
```

Reply:
Of course, thanks for the hints.